MAINTAINERS_GUIDE.md

```diff
@@ -29,7 +29,7 @@ It is every maintainer's responsibility to:
 * 4) Make sure their component respects the philosophy, design and
   roadmap of the project.
 
-## How are decisions made?
+## The method of making decisions
```

Comment:
:smile:
It could be a good choice that "How to make decisions".
It is a difficult to make the sentence smoth for us.